docs/source/install/upgrades.rst

```diff
@@ -170,10 +175,10 @@ any skipped versions:
 v3.4
 ''''
 
-*  |st2| now uses python 3 on Ubuntu 16 and RHEL/CentOS 7. Therefore any packs that only support python 2 will need to be upgraded to python 3.
+*  |st2| now uses python 3 on Ubuntu 16 and RHEL/CentOS 7. Therefore any packs that only support python 2 will need to be upgraded to python 3. All packs installed prior to upgrade on Ubuntu 16 and RHEL/CentOS 7 will need to have their virtualenvironment re-created after upgrade, using the ``packs.setup_virtualenv`` action.
```

Comment:
Maybe we can also document / link to the ``sudo st2ctl reload --register-setup-virtualenvs`` option (that's the one I assume in "gitops" / infra as code manner).